_fluo-docs-1-2/getting-started/design.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Design
+category: getting-started
+order: 1
+---
+
+For a general overview of Fluo, take the [Fluo tour](/tour/). The diagram below provides an
```

Comment:
Not sure this link to Fluo tour is needed, but it doesn't hurt.

Reply:
Removed the link.